### PR TITLE
add first pass at website

### PR DESCRIPTION
**Pull request by ryanatkn**
swaps domains with ui.felt.dev here: https://github.com/feltjs/felt-ui/pull/273

### CODE REVIEW

File: src/lib/project.ts [hunk 1]
```diff
@@ -1,15 +1,69 @@
...
 		'transparency and accountability',
 		'simplicity and efficiency',
-		'easy and inexpensive self-hosting',
+		'easy and low-resource-usage self-hosting',
 		'privacy by default',
 		'accessibility, inclusivity, and diversity',
-		'going with the grain of the web',
 		'fostering learning, innovation, and experimentation',
 	],
 };
```
Comment (ryanatkn) [inline src/lib/project.ts:69]: tweaked the values a bit

File: src/lib/project.ts [hunk 2]
```diff
@@ -1,15 +1,79 @@
...
+			source: 'https://github.com/feltjs/util',
+		},
+	],
 	values: [
 		'we put the people who use our software first',
 		'sustainability through aligned incentives',
 		'intentional collaboration, community, and governance',
-		'open source and interoperability',
+		'open source',
+		'interoperability and web-friendliness',
```
Comment (greatbacon) [inline src/lib/project.ts:71]: I still think this and open-source could get away with being one line

File: src/lib/project.ts [hunk 3]
```diff
@@ -1,15 +1,79 @@
...
 		'transparency and accountability',
 		'simplicity and efficiency',
-		'easy and inexpensive self-hosting',
+		'easy and low-resource-usage self-hosting',
```
Comment (greatbacon) [inline src/lib/project.ts:74]: maybe "efficient"? low-resource-usage feels to technical 😐

File: src/lib/project.ts (hunk 2 above)
Comment (ryanatkn) [inline src/lib/project.ts:71]:
```suggestion
		'open source and interoperability',
```

Comment (ryanatkn) [reply src/lib/project.ts]: k, combined and deleted the web-friendliness

File: src/lib/project.ts (hunk 3 above)
Comment (ryanatkn) [reply src/lib/project.ts:73]: things can be efficient but expensive, was trying to fix if you didn't like inexpensive, I'll revert unless there's something more accurate

File: src/lib/project.ts [hunk 4]
```diff
@@ -1,15 +1,78 @@
...
+	],
 	values: [
 		'we put the people who use our software first',
 		'sustainability through aligned incentives',
 		'intentional collaboration, community, and governance',
 		'open source and interoperability',
 		'transparency and accountability',
 		'simplicity and efficiency',
-		'easy and inexpensive self-hosting',
+		'easy and low-resource-usage self-hosting',
```
Comment (ryanatkn) [inline src/lib/project.ts:73]:
```suggestion
		'easy and inexpensive self-hosting',
```

**Review by greatbacon — APPROVED** [review]

**Pull request merged**